### PR TITLE
reorganization of examples

### PR DESCRIPTION
**Pull request by jones-michael-s**
This PR implements a reorganization of the aa example scripts (see Discussion #327). It's mostly just a bunch of file renaming to make the example script names simpler and consistent, but there's two small code changes:

1) I added a new script create_basic_parameterfile.m that replaces the (now deleted) first tutorial. Use of this utility will be discussed in the aa tutorial that I will post in the docs.

2) I modified testExamples.m to reflect the new examples/tutorials organization. The change *seems* straightforward but this is code I'm not very familiar with so feel free to check my work. 

The new version of the tutorial doesn't play nice with automated testing so I kept a copy of the original for testing which now lives in  developer/testscripts/tutorials. That's a bit hacky, but I couldn't see how to combine the versions without making the tutorial content (more) confusing to the new user.

### CODE REVIEW

**Review by tiborauer — APPROVED** [review]

**Comment by tiborauer** [comment]
It seems they slightly reduced the hard disks on the runners. For the CI to run, you may need to reduce the requested disk size.. You can try changing all lines in the YML files containing "root-reserve-mb: 50176" to  "root-reserve-mb: 49152", i.e. requesting 48 GB instead of 49GB.

**Comment by jones-michael-s** [comment]
Updated the yml and meeg scripts as we discussed and the CI is still failing. I looked at the git diffs on these files and I *think* I changed them correctly. Any thoughts?

**Comment by jones-michael-s** [comment]
FYI: I created a branch on my fork to play with titrating the requested disk space -- cutting it in half (to 25088) gets the CI  past the fallocate crash. It's currently cranking though the testscripts but given what you've mentioned about the disk space requirements of some of them I'm not expecting the CI to complete...

TL;DR: It does seem to be a disk space request issue, but the magic number appears to be less than 49152.

**Comment by jones-michael-s** [comment]
Further CI testing on my fork indicates 48128 might work. Going to give that a try here....
